src/Illuminate/Foundation/Console/ObserverMakeCommand.php

```diff
@@ -69,11 +69,9 @@ protected function replaceModel($stub, $model)
 
         $namespaceModel = $this->laravel->getNamespace().$model;
 
-        if (Str::startsWith($model, '\\')) {
-            $stub = str_replace('NamespacedDummyModel', trim($model, '\\'), $stub);
-        } else {
-            $stub = str_replace('NamespacedDummyModel', $namespaceModel, $stub);
-        }
+        $stub = Str::startsWith($model, '\\')
```

Comment:
How is this any better than the IFs? I find `if(){ ... }else{ ... }` much clearer.